metron-streaming/Metron-DataLoads/pom.xml

```diff
@@ -149,7 +149,12 @@
             <version>${project.parent.version}</version>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>1.10.16</version>
```

Comment:
Ryan's PR from last week includes mockito as well and I believe there's a global mockito variable.  When you get around to merging master (after ryan's PR is merged), please make sure that this is referring to the global mockito version.